### ISSUE TITLE
unable to access webapp client from mobile

### ISSUE THREAD

**Issue by amenak77** [freedomotic/fd-vue-webapp#135]
![test_newclient4](https://user-images.githubusercontent.com/13914556/43365301-0142b360-932b-11e8-9a39-cecdf14c0409.jpg)

After compiling webapp client's project and replacing default polymer client with it into webserver plugin, webapp is correctly found at [server IP address]:8090 address; if you try to access from a pc or notebook within the LAN there is no problem. If you try from a smartphone you have a rightful login panel which deny access even if you enter right data. Ping is sent and received correctly; this probably means an auth problem more than a connection one. Mobile problem only.

**Comment by mcicolella** [comment]
Is the pc or the notebook the same which FD is running on when it works?

**Comment by amenak77** [comment]
Yes, when it works, it is the same notebook and the call is made through localhost; I haven't tried access to freedomotic via webapp from another LAN pc; I'll try ASAP; however, I downloaded firefox mobile app and used it to access freedomotic and I have same bad login message

**Comment by amenak77** [comment]
Hi all,
I tried to access freedomotic from another LAN computer thriught webapp client; first step is OK: you digit in browswer (chrome) navbar 192.168.1.58:8090 and apperas vue login; then if you enter Fd notebook ip address into server address field, 9111 in server address field plus ususal admin/admin you have THE SAME BAD LOGIN INFORMATION warning; so I think we can change the issue in "unable to access webapp client from another LAN device". PS using localhost (obviously) same failure. Sometimes...

**Comment by amenak77** [comment]
Ever and ever strange; if you access freedomotic from the same LAN device, access is granted whatever charachter you digit into the first thwo form fields (server address and server port) if you correcly insert admin/admin; otherwise access is denied. The last issue definition should be "errors with auth LAN token ring" I suppose

**Comment by mcicolella** [comment]
The problem is [here](https://github.com/freedomotic/fd-vue-webapp/blob/e702ab67775e90ba5e3cd10232bda0b36555e2a9/src/utils/fd-axios.js#L11).
RestAPI url is set statically.

**Comment by amenak77** [comment]
Excuse me, but I don't understand; why is this a problem?

**Comment by mcicolella** [comment]
Because it points always to 'localhost'. If the client is on the same machine running FD all is OK.
When you connect from another device (a smartphone for example) the app searches RestAPI on the 'localhost' that isn't the PC where the plugin is running but the client itself. So it doesn't find the API.
Hope it's clear.

**Comment by amenak77** [comment]
Perfectly clear, thank you; so we can get rid of this annoying bug soon ;)

**Comment by mcicolella** [comment]
Just updated the app. The issue should be fixed.
I removed server ip and port number because the first is automatically detected from the URL and the last is set statically on the RestAPI. Less parameters for users :)
Please check. 
N.D. remember to delete the browser history

**Comment by amenak77** [comment]
Great! I'll check ASAP and obviously I'll let you know ;)

**Comment by amenak77** [comment]
First of all, just to make my point clear: fantastic work! Great Mauro and great Ubaldo. All works very well; the webapp has a great level and personally, command my home lights without install or configure anything. Autodiscovery at its maximum extent; even ip address and restapi port don't reqire any configuration. UI is set automatically depending on type of device you have. Fantastic
